src/main/kotlin/mate/academy/Ball.kt

```diff
@@ -0,0 +1,33 @@
+package mate.academy
+
+class Ball {
+    private var ballColor: String = ""
+    private var ballNumber: Int = 0
```

Comment:
No need to set default value, as you are always changing it when initing new object